src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/UpdateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,223 @@
+@ignore
+Feature: Resources "Update" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify that existing resources can be updated successfully
+            # The id value should be replaced with the resource created in the Background section
+             When a PUT request is made to "/ed-fi/absenceEventCategoryDescriptors/<id>" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 204
+
+        @ignore
+        Scenario: Verify updating a resource with valid data
+             # The id value should be replaced with the resource created in the Background section
+             When a PUT request is made to "/ed-fi/absenceEventCategoryDescriptors/<id>" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 204
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify error handling updating a resource with invalid data
+            # The id value should be replaced with the resource created in the Background section
+             When a PUT request is made to "/ed-fi/absenceEventCategoryDescriptors/<id>" with
```

Comment:
For consistency, please use `{id}` rather than `<id>`.

Reply:
Replaced in the Resources Folder